### PR TITLE
Updating the header and file name per OSS CELA recommendation.

### PR DESCRIPTION
**Pull request by msjasteppe**
Updating the file name and header per OSS CELA recommendation.

### CODE REVIEW

**Comment by rbhaiya** [comment]
Looks like standard license.

File: NOTICE.md [hunk 1]
```diff
@@ -1,541 +1,539 @@
...
-> 
-> Exhibit B - "Incompatible With Secondary Licenses" Notice
-> 
->       This Source Code Form is "Incompatible
->       With Secondary Licenses", as defined by
+# NOTICES
+
+This repository incorporates material as listed below or described in the code.
+
+## Ensure.That 9.2.0
```
Comment (rogordon01) [inline NOTICE.md:5]: Do we still need to list all of these individual software licenses?

Comment (msjasteppe) [reply NOTICE.md:5]: Yes. This is just a file name change and an adjustment to the header per this documentation - https://docs.opensource.microsoft.com/using/required-notice-template/

Comment (msjasteppe) [reply NOTICE.md:5]: @rogordon01 @rbhaiya Based on our conversations yesterday, it appears that we no longer need this file. Do we want to shutdown this PR, archive a copy of the file and then remove it completely?

Comment (rogordon01) [reply NOTICE.md:5]: The PR can be abandoned for now. I looked yesterday and all of the software listed here _seems_ to be pulled in as a reference. We can confirm with @dustinburson and @namalu  when they come back. At which point we can issue a new PR to delete the file.
The file will always be in a our Git history, so no need to manual archive it.

**Comment by msjasteppe** [comment]
Closing PR per Rob's recommendation. We will pick this up once Dustin and Nate are back in the office.

**Pull request closed without merging**